Formula/s/simple-tiles.rb

```diff
@@ -20,14 +20,14 @@ class SimpleTiles < Formula
   end
 
   depends_on "pkg-config" => [:build, :test]
+  depends_on "python@3.11" => :build
   depends_on "cairo"
   depends_on "gdal"
   depends_on "pango"
 
-  uses_from_macos "python" => :build
-
   def install
-    system "python3", "./waf", "configure", "--prefix=#{prefix}"
+    ENV.prepend_path "PATH", Formula["python@3.11"].libexec/"bin"
+    system "./waf", "configure", "--prefix=#{prefix}"
```

Comment:
Better to pass the path to the actual executable you want to use:
```suggestion
    python_exe = Formula["python@3.11"].opt_bin/"python3.11"
    system python_exe, "./waf", "configure", "--prefix=#{prefix}"
```

Reply:
That unfortunately does not work, the issue is the project's `Makefile` in the next step also calls `waf` and it uses shebang `/usr/bin/env python`.

Reply:
Gotcha, that's reasonable 👍